.github/workflows/image-scan.yml

```
@@ -0,0 +1,23 @@
+name: Scan and Generate Grype Report
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)